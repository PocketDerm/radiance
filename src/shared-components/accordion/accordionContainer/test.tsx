import React from 'react';
import renderer from 'react-test-renderer';

import AccordionContainer from './index';

describe('<AccordionContainer />', () => {
  test('Renders with styles by default', () => {
    const component = renderer.create(
      <AccordionContainer>I am the container child</AccordionContainer>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('Renders without styles if explicit falsy withContainer prop', () => {
    const component = renderer.create(
      <AccordionContainer withContainer={false}>
        I am the fragment child
      </AccordionContainer>,
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
