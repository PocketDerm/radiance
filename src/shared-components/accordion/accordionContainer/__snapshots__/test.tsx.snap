// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccordionContainer /> Renders with styles by default 1`] = `
.emotion-0 {
  box-shadow: 0px 8px 24px rgba(52,51,82,0.05);
  background-color: #ffffff;
  max-width: 768px;
}

<div
  className="emotion-0 emotion-1"
>
  I am the container child
</div>
`;

exports[`<AccordionContainer /> Renders without styles if explicit falsy withContainer prop 1`] = `"I am the fragment child"`;
